### DISCUSSION TITLE
Return to menu after scanning and reporting spammers comments

### DISCUSSION THREAD

**Discussion by SeaN0X** [ThioJoe/YT-Spammer-Purge#204] [Ideas]
I would really be grateful if after scanning and reporting spammers, the program would go back to its main menu so that I could do other scans instead of just closing it. As my computer is very slow, when I run the program it takes about 40 seconds for the menu to appear.

This request is not really something necessary but it would help save some time and have a better interface in my opinion :)

**Comment by Abelkrijgtalles** [comment]
You could add also question if people want to quit or not

**Comment by robeya123** [comment]
I came on here to make the same suggestion. Does the application need to Quit each time? Maybe add the option to the config file as well so we can have a default response?

**Comment by TechStudent10** [comment]
#259 I added this feature.
